R/get_units.R

```diff
@@ -89,115 +89,116 @@
+#'          attribute.
+#'        \item \code{class}: The named economic attribute class (e.g.,
+#'          "precious commodity").
+#'      }
+#'    \item \code{measure}: a \code{dataframe} containing the measure
 #'      attributes present within the unit, with the following columns:
-#'   \itemize{
-#'      \item \code{measure_class}: The class of measures (e.g. "geochemical").
-#'      \item \code{measure_type}: the type of measures (e.g. "minor
-#'      elements").
-#'   }}
-#'   \itemize{
+#'      \itemize{
+#'        \item \code{measure_class}: The class of measures (e.g.,
+#'          "geochemical").
+#'        \item \code{measure_type}: the type of measures (e.g., "minor
+#'          elements").
+#'      }
 #'    \item \code{notes}: Unit specific notes.
 #'    \item \code{color}: Recommended coloring for units based on dominant
-#'    lithology.
+#'      lithology.
 #'    \item \code{text_color}: Recommended coloring for text based on color.
 #'    \item \code{t_int_id}: The ID of the chronostratigraphic interval
-#'    containing the top boundary of the unit.
+#'      containing the top boundary of the unit.
 #'    \item \code{t_int_name}: The name of the time interval represented at
-#'    the top of the unit.
+#'      the top of the unit.
 #'    \item \code{t_int_age}: The age of the chronostratigraphic interval
-#'    containing the top boundary of the unit.
+#'      containing the top boundary of the unit.
 #'    \item \code{t_prop}: Position of continuous time age model top boundary,
-#'    proportional to reference time interval (t_interval).
+#'      proportional to reference time interval (t_int_name).
 #'    \item \code{units_above}: The unit_ids of the units contacting the top
-#'    of the unit.
+#'      of the unit.
 #'    \item \code{b_int_id}: The ID of the chronostratigraphic interval
-#'    containing the bottom boundary of the unit.
+#'      containing the bottom boundary of the unit.
 #'    \item \code{b_int_name}: The name of the time interval represented at
-#'    the
-#'    bottom of the unit.
+#'      the bottom of the unit.
 #'    \item \code{b_int_age}: The age of the chronostratigraphic interval
-#'    containing the bottom boundary of the unit.
+#'      containing the bottom boundary of the unit.
 #'    \item \code{b_prop}: Position of continuous time age model bottom
-#'    boundary, proportional to reference time interval (b_interval).
+#'      boundary, proportional to reference time interval (b_interval).
 #'    \item \code{units_below}: The unit_ids of the units contacting the
-#'    bottom of the unit.
+#'      bottom of the unit.
 #'    \item \code{strat_name_long}: The stratigraphic name associated with
-#'    the unit (e.g. "Coal Valley Formation").
-#'    \item \code{refs}: The number of references associated with the unit.
+#'      the unit (e.g., "Coal Valley Formation").
+#'    \item \code{refs}: The IDs of the references associated with the unit.
```

Comment:
In a lot of the other documentation we have used the unique identification number rather than ID (just for consistency).